artifactory/commands/golang/go.go

```diff
@@ -139,6 +167,41 @@ func (gc *GoCommand) Run() error {
 
 	return err
 }
+func copyGoPackageFiles(destPath, packageName, rtTargetRepo string, rtDetails auth.ServiceDetails) error {
+	var packageFilesPath, version string
+	packageCachePath, err := executersutils.GetPackagePath()
+	if err != nil {
+		return err
+	}
+	packageNameSplitted := strings.Split(packageName, "@")
+	// The Case the user ask for specifc version
+	if len(packageNameSplitted) == 2 && strings.HasSuffix(packageNameSplitted[1], "v") {
+		version = packageNameSplitted[1]
+	} else {
+		// find the version using Artifactory
+		packageName = filepath.Join(packageNameSplitted[0], "@v")
+		if len(packageNameSplitted) == 1 {
+			// no version was given to get command, so the latest was downloaded
```

Comment:
// no version was given to get command, so the latest was downloaded -->
// No version was given to the "go get" command, so the latest version should be downloaded